tutorials/advanced/transforms_tutorial.py

```diff
@@ -3,80 +3,94 @@
-|                |                             |bottom left of the figure, and     |
-|                |                             |(width, height) is the top right   |
-|                |                             |of the figure in inches.           |
-+----------------+-----------------------------+-----------------------------------+
-|"display"       |``None``, or                 |The pixel coordinate system of the |
-|                |``IdentityTransform()``      |display window; (0, 0) is bottom   |
-|                |                             |left of the window, and (width,    |
-|                |                             |height) is top right of the        |
-|                |                             |display window in pixels.          |
-+----------------+-----------------------------+-----------------------------------+
-|"xaxis",        |``ax.get_xaxis_transform()``,|Blended coordinate systems; use    |
-|"yaxis"         |``ax.get_yaxis_transform()`` |data coordinates on one of the axis|
-|                |                             |and axes coordinates on the other. |
-+----------------+-----------------------------+-----------------------------------+
-
-All of the transformation objects in the table above take inputs in
-their coordinate system, and transform the input to the *display*
-coordinate system.  That is why the *display* coordinate system has
-``None`` for the ``Transformation Object`` column -- it already is in
-*display* coordinates.  The transformations also know how to invert
-themselves, to go from *display* back to the native coordinate system.
-This is particularly useful when processing events from the user
-interface, which typically occur in display space, and you want to
-know where the mouse click or key-press occurred in your *data*
-coordinate system.
-
-Note that specifying objects in *display* coordinates will change their
-location if the ``dpi`` of the figure changes.  This can cause confusion when
-printing or changing screen resolution, because the object can change location
-and size.  Therefore it is most common
-for artists placed in an Axes or figure to have their transform set to
-something *other* than the `~.transforms.IdentityTransform()`; the default when
-an artist is placed on an Axes using `~.axes.Axes.add_artist` is for the
-transform to be ``ax.transData``.
++----------------+-----------------------------------+-----------------------------+
+|Coordinate      |Description                        |Transformation object        |
+|system          |                                   |from system to display       |
++================+===================================+=============================+
+|"data"          |The coordinate system for the data,|``ax.transData``             |
+|                |controlled by xlim and ylim.       |                             |
++----------------+-----------------------------------+-----------------------------+
```

Comment:
```suggestion
|"data"          |The axes coordinate system for the data,|``ax.transData``             |
|                |controlled by xlim and ylim.       |                             |
+----------------+-----------------------------------+-----------------------------+
```

Reply:
I would say 'The coordinate system for the data in an Axes'